scripts/includes/rhel.sh

```diff
@@ -9,35 +9,42 @@ install_yum_utils() {
 get_full_pkg_versions() {
   if [ "$VERSION" != '' ];
   then
-    local ST2_VER=$(repoquery -y --nvr --show-duplicates st2 | grep -F st2-${VERSION} | sort --version-sort | tail -n 1)
+    local RHMAJVER=`cat /etc/redhat-release | sed 's/[^0-9.]*\([0-9.]\).*/\1/'`
+    local YES_FLAG=""
+    if [ "$RHMAJVER" -ge "8" ]; then
+      # RHEL 8 and newer, you need "-y" flag to avoid being prompted to confirm "yes"
+      local YES_FLAG="-y"
+    fi
+
+    local ST2_VER=$(repoquery ${YES_FLAG} --nvr --show-duplicates st2 | grep -F st2-${VERSION} | sort --version-sort | tail -n 1)
     if [ -z "$ST2_VER" ]; then
       echo "Could not find requested version of st2!!!"
-      sudo repoquery -y --nvr --show-duplicates st2
+      sudo repoquery ${YES_FLAG} --nvr --show-duplicates st2
       exit 3
     fi
     ST2_PKG=${ST2_VER}
 
-    local RHMAJVER=`cat /etc/redhat-release | sed 's/[^0-9.]*\([0-9.]\).*/\1/'`
-    local ST2MISTRAL_VER=$(repoquery -y --nvr --show-duplicates st2mistral | grep -F st2mistral-${VERSION} | sort --version-sort | tail -n 1)
-    if [ -z "$ST2MISTRAL_VER" && "$RHMAJVER" != '8' ]; then
+    local ST2MISTRAL_VER=$(repoquery ${YES_FLAG} --nvr --show-duplicates st2mistral | grep -F st2mistral-${VERSION} | sort --version-sort | tail -n 1)
+    # RHEL 8 and newer does not install Mistral
+    if [ -z "$ST2MISTRAL_VER" -a "$RHMAJVER" -lt "8" ]; then
```

Comment:
```suggestion
    if [ -z "$ST2MISTRAL_VER" && "$RHMAJVER" -lt "8" ]; then
```

Please rely on a more straightforward `&&` which is what we use everywhere. `-a` may lead to confusion, see https://stackoverflow.com/a/321493/4533625